content/releases-and-versions/versions.md

```diff
@@ -1,13 +1,13 @@
 ---
 description: A list of tools available out-of-the-box on Codemagic build machines.
 title: macOS build machine specification (Xcode 11.x)
-weight: 3
+weight: 2
 ---
 
 ## Hardware
 
-- Standard VM on Mac mini `2.3GHz Quad Core / 8GB`
-- Premium VM on Mac Pro `3.7GHz Quad Core / 32GB`
+- Mac Mini `2.3GHz Quad Core / 8GB`
+- Mac Pro `3.7GHz Quad Core / 32GB`

```

Comment:
please change the name of the machines so it's in accordance with what we have on our pricing page